jupyter_scheduler/scheduler.py

```diff
@@ -608,6 +613,17 @@ def list_job_definitions(self, query: ListJobDefinitionsQuery) -> ListJobDefinit
 
         return list_response
 
+    def create_job_from_definition(self, job_definition_id: str, model: CreateJobFromDefinition):
+        job_id = None
+        definition = self.get_job_definition(job_definition_id)
+        if definition and definition.active:
+            input_uri = self.get_staging_paths(definition)["input"]
```

Comment:
Does this end up copying the input file to the staging directory for the job?

Reply:
Yes, this references the create_job method which copies the file.